src/js/views/Certificates/layout/AssociateToDeviceModal.jsx

```diff
@@ -112,12 +114,14 @@ const AssociateDevicesModal = ({ isOpen, certificate, handleHideDevicesToAssocia
 
   return (
     <Dialog open={isOpen} onClose={handleHideDevicesToAssociateModal} maxWidth='lg' fullWidth>
+      <Helmet title={`${t('certificates:associateToDevice')} • ${t('common:dojotPageTitle')}`} />
       <DialogHeader
-        title={t('AssociateToDeviceModalTitle')}
+        title={t('certificates:associateToDevice')}
         handleHideDialog={handleHideDevicesToAssociateModal}
       />
 
       <Box padding={2}>
+        <Typography>{t('certificates:dataTableTitle')}</Typography>
```

Comment:
Consider wrapping this text with a Box to add margin bottom or create a className and do the same with CSS.